### DISCUSSION TITLE
Editar encabezado «Cursos más populares»

### DISCUSSION THREAD

**Discussion by jolufe** [chamilo/chamilo-lms#5970] [General] [answered]
Hola.

¿Cómo puedo editar/eliminar el encabezado «Cursos más populares»?

SO: Windows 10
Chamilo: 1.11.28

![screenshot](https://github.com/user-attachments/assets/4b34df61-7e1e-4200-b5a6-8d1938ed3daf)

**Comment by chatonito** [comment, marked as answer]
utilice un subidioma, puede cambiarlo ahí y poner el título que le agrade, nosotros lo tenemos como "cursos más visitados".
saludos

**Reply by jolufe** [reply]
Muchas gracias @chatonito 

Sañludos.
